components/brave_new_tab_ui/components/default/braveToday/options/customize.tsx

```diff
@@ -6,48 +6,27 @@
 import * as React from 'react'
 import styled from 'styled-components'
 import { getLocale } from '../../../../../common/locale'
+import { Button } from '../default'
 
 type Props = {
   show: boolean
   onCustomizeBraveToday: () => any
 }
 
-const CustomizeButton = styled('button')<Props>`
+const Hideable = styled('div')<Props>`
```

Comment:
Suggestion: How about fixing https://github.com/brave/brave-browser/issues/15221 together because you touches Customize button herre. After it's hidden, still we can click. Of course, I'm fine as a separate PR.

Reply:
Good idea. I hadn't noticed that PR.